image/templates/helm/stackrox-secured-cluster/templates/collector.yaml

```diff
@@ -147,6 +147,28 @@ spec:
           name: tmp-volume
         - mountPath: /cache
           name: cache-volume
+      - name: node-inventory
+        image: {{ quote ._rox.image.scanner.fullRef }}
+        imagePullPolicy: IfNotPresent
+        command: ["/scanner", "--nodeinventory", "--config=", ""]
+        ports:
+          - containerPort: 8444
+            name: grpc
+        env:
+        - name: ROX_NODE_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        {{- include "srox.envVars" (list . "daemonset" "collector" "node-inventory") | nindent 8 }}
+        volumeMounts:
+         - mountPath: /host
+           name: host-root-ro
+           readOnly: true
+         - mountPath: /tmp/
+           name: tmp-volume
+         - mountPath: /cache
+           name: cache-volume
```

Comment:
[No change needed here]
These two mounts are exclusively needed by the Node Scanning code.
As we moved that into this image, we could remove the mounts from Compliance (L#146-149). 
As they don't hurt, we can also leave them for now. Up to you.

Reply:
Oh, good point, we should definitely clean this up